### DISCUSSION TITLE
Specific class clothes inventory

### DISCUSSION THREAD

**Discussion by BossCakes21** [FakeFishGames/Barotrauma#4768] [Ideas and suggestions]
I think each class should have a bit of uniqueness, like how the medic class, it's clothes has 3 slots for medical items, why not for the other classes. like 
1. the mechanic could have more welding fuel and tools in general
2. engineer could have wires and components, maybe batteries 
3. security could have more ammo space and hold small arms
4. captain could have more everything he is the captain
5. the assistant could just have a slight increase to run speed as he is just wearing PJ's basically
Another thing would this make some clothing more used then others, so would it only work if its the starting ones, or if you are the class you get that benefit, or just make it so you have to craft the pockets on to the clothing. 
Would love to see what others think had this idea for a while.

**Comment by Saevon** [comment]
See #4734 for a similar idea, except replacing the toolbelt

**Reply by BossCakes21** [reply]
> 
> 
> See #4734 for a similar idea, except replacing the toolbelt

I have seen that post, but I would like to see more unique class related things. because right now in my opinion i feel like each class is the same but with different skill levels.
